app/controllers/sinicum/controller_base.rb

```diff
@@ -106,6 +106,9 @@ def redirect_redirect_page
         redirect_target = page[:redirect_link] || page[:external_redirect_link]
         if Sinicum::Util.is_a_uuid?(redirect_target)
           redirect_target = Sinicum::Jcr::Node.find_by_uuid("website", redirect_target).try(:path)
+          if redirect_target && page[:anchor].present?
```

Comment:
Should this not move below the `Sinicum::Util.is_a_uuid?` conditional? Should the anchor not be added to the link regardless if it is an internal redirect or an external?

Otherwise, I would assume that it would be confusing if someone enters a value in the `anchor` field an sees that it has no effect.

Reply:
The anchor property is for the moment only applied for internal links in case of JCR-Node, since the external link could already have it in its characters string.